Mathlib/LinearAlgebra/Matrix/Zpow.lean

```diff
@@ -123,25 +123,25 @@ theorem zpow_neg_coe_nat (A : M) (n : ℕ) : A ^ (-n : ℤ) = (A ^ n)⁻¹ := by
 #align matrix.zpow_neg_coe_nat Matrix.zpow_neg_coe_nat
 
 theorem IsUnit.det_zpow {A : M} (h : IsUnit A.det) (n : ℤ) : IsUnit (A ^ n).det := by
-  cases n
+  cases' n with n n
   · simpa using h.pow n
   · simpa using h.pow n.succ
-#align is_unit.det_zpow IsUnit.det_zpow
+#align is_unit.det_zpow Matrix.IsUnit.det_zpow
```

Comment:
This change is incorrect. You should almost never edit an `#align`. You need to fix the name of the lemma instead.